src/python/grpcio_tests/tests/protoc_plugin/beta_python_plugin_test.py

```diff
@@ -657,6 +666,23 @@ def half_duplex_request_iterator():
                 with self.assertRaises(face.ExpirationError):
                     next(responses)
 
+    def testRelativeImport(self):
+        self._protoc()
+
+        # Our invocation of `_protoc` above will cause the
+        # test_relative_grpc_pb2 file to contain a line that looks like:
+        #
+        # `from . import test_relative_pb2 as _dot_test__relative__pb2`
+        #
+        # We can check to ensure that the GetFooRequest has been imported
+        # to the gRPC Python module by reading through the grpc file
+        # into the raw protobuf file.
+        #
+        # If the line doesn't raise an exception (specifically, an
+        # AttributeError), we know the import worked, and we need not further
+        # assert anything.
+        self._relative_pb2._dot_test__relative__pb2.GetFooRequest()
```

Comment:
This is a little too whitebox. The symbol `_dot_test__relative__pb2` is an implementation detail. [Hyrum's Law](https://www.hyrumslaw.com/) might bite us if we ever try to do it, but we reserve the right to change the name with which we import transitive protos.